doc_admin/AdminManual.adoc

```diff
@@ -21,9 +21,13 @@ pass:[<br>][.small]#Policies is made available under a http://creativecommons.or
 
 == 1 Abstract
 
-This application is the first of a set of software products that ISFfootnote:[Informatici Senza Frontiere = Computer scientists without frontiers] has developed to support the management and the activities of the St. Luke Hospital in Angal (Uganda). After that initial mission, the St. Luke Hospital has become the first one of a long list of hospitals that have found this software useful.
+[Open Hospital][openhospital] (OH) is a free and open-sourcefootnote:[Open-source = software for which the original *source code* is made available to anyone free of charge.] Electronic Health Record (EHR) application.
+Open Hospital is deployed as desktop application that can be used in a standalone (portable), single user mode
```

Comment:
`deployed as desktop application` should be `deployed as a desktop application`

Reply:
Fixed